airflow/www/templates/airflow/grid.html

```diff
@@ -42,6 +42,7 @@
     const stateColors = {{ state_color_mapping|tojson }};
     const autoRefreshInterval = {{ auto_refresh_interval }};
     const defaultDagRunDisplayNumber = {{ default_dag_run_display_number }};
+    const defaultWrap = {{ default_wrap|tojson }};
```

Comment:
Does `tojson` convert to boolean?

Reply:
Yes, I get an error without it, he is trying to access a variable named `True`

Reply:
![image](https://user-images.githubusercontent.com/14861206/184737585-d17c10ed-5fbd-433a-b7e3-d70131bdb485.png)

Reply:
I could replace this with something more explicit:
```
    const defaultWrap = {{ 'true' if default_wrap else 'false' }};
```

Reply:
Actually, we should put this in a `<meta>` tag vs directly accessing here. and then in the js file we can call `const defaultWrap = getMetaValue('default_wrap') === 'True';`

Reply:
Done :)